includes/widgets/class-wc-widget-rating-filter.php

```diff
@@ -97,29 +97,33 @@ public function widget( $args, $instance ) {
 
 		ob_start();
 
-		$found         = false;
-		$rating_filter = isset( $_GET['rating_filter'] ) ? array_filter( array_map( 'absint', explode( ',', wp_unslash( $_GET['rating_filter'] ) ) ) ) : array(); // WPCS: input var ok, CSRF ok, sanitization ok.
+		$found = false;
+
+		$_GET['rating_filter'] = isset( $_GET['rating_filter'] ) ? absint( wp_unslash( $_GET['rating_filter'] ) ) : 0; // WPCS: input var ok, CSRF ok, sanitization ok.
```

Comment:
I don't think setting global properties like this is considered best practice. Why not keep the local variable?